app/src/main/java/ch/epfl/sweng/SDP/game/VotingPageActivity.java

```diff
@@ -309,8 +366,10 @@ public void onDataChange(@NonNull DataSnapshot dataSnapshot) {
                             // Get the image from the local database instead
                             LocalDbHandlerForImages localDbHandler = new LocalDbHandlerForImages(
                                     getApplicationContext(), null, 1);
-                            storeBitmap(localDbHandler.getLatestBitmapFromDb(), currentId);
+                            Bitmap localImage = localDbHandler.getLatestBitmapFromDb();
```

Comment:
There is really no need to create a variable for that I think

Reply:
Why is it resolved if you haven't even answered

Reply:
I fixed it ;)

Reply:
Again